src/cmd/bakeversion.rs

```diff
@@ -7,28 +7,33 @@ use memmap2::MmapOptions;
 
 #[derive(FromArgs, PartialEq, Eq, Debug)]
 /// Sets the MetroidBuildInfo tag value in a given binary.
-#[argp(subcommand, name = "metroidbuildinfo")]
+#[argp(subcommand, name = "bakeversion")]
 pub struct Args {
     #[argp(positional)]
     /// path to source binary
     binary: PathBuf,
     #[argp(positional)]
+    /// build tag marker to search for
+    build_tag: String,
+    #[argp(positional)]
     /// path to build info string
     build_info: PathBuf,
+    #[argp(option, short = 'm', default="35")]
+    /// the max length of the version string, excluding build_tag (default: 35)
+    max_len: usize,
```

Comment:
I wonder if we should search for the replacement text instead, and then max_len would be inferred. I feel like this would be more generic.

Reply:
Though I’m not sure how this will work with dtk dol split, since it uses the original DOL data. Maybe we need a way to replace a symbol’s value?

Reply:
I actually suggested adding a method to override a value for this exact purpose in the prime decomp channel.